core/templates/dev/head/pages/exploration_editor/ExplorationEditor.ts

```diff
@@ -233,6 +236,18 @@ oppia.controller('ExplorationEditor', [
     $scope.revertExplorationUrl = (
       '/createhandler/revert/' + $scope.explorationId);
 
+    var setPageTitle = function() {
+      if (ExplorationTitleService.savedMemento) {
+        PageTitleService.setPageTitle(
+          ExplorationTitleService.savedMemento + ' - Oppia Editor');
+      } else {
+        PageTitleService.setPageTitle(
+          'Untitled Exploration - Oppia Editor');
+      }
+    };
+
+    $scope.$on('explorationPropertyChanged', setPageTitle);
```

Comment:
Can you please save `'explorationPropertyChanged'` into angular constant.

Reply:
Okay

Reply:
Done